package/pruners/hyperband/README.md

```diff
@@ -0,0 +1,57 @@
+---
+author: Optuna team
+title: Hyperband Pruner
+description: Pruner using Hyperband.
+tags: [pruner, built-in]
+optuna_versions: [3.6.1]
+license: MIT License
+---
+
+## Class or Function Names
+
+- HyperbandPruner
+
+## Example
+
+```python
```

Comment:
I think it would be great to split examples into executable file, since it enables lints and makes users to run it easily.